configs/default/TMTR-TMOTORVELOXF7V2.config

```diff
@@ -0,0 +1,124 @@
+# Betaflight / TMOTORVELOXF7V2 (TMR7) 4.3.1 Sep 23 2022 / 17:19:06 () MSP API: 1.44
+
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
+#define USE_FLASH_M25P16
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
```

Comment:
Please remove MAG - if not physical present on board

Reply:
I have removed it.